homeassistant/components/counter/__init__.py

```diff
@@ -31,22 +34,33 @@
 SERVICE_DECREMENT = 'decrement'
 SERVICE_INCREMENT = 'increment'
 SERVICE_RESET = 'reset'
+SERVICE_SETUP = 'setup'
 
-SERVICE_SCHEMA = vol.Schema({
-    vol.Optional(ATTR_ENTITY_ID): cv.comp_entity_ids,
+SERVICE_SCHEMA_SIMPLE = vol.Schema({
+    vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
+})
+SERVICE_SCHEMA_SETUP = vol.Schema({
+    ATTR_ENTITY_ID: cv.entity_ids,
+    vol.Optional(CONF_MINIMUM): vol.Any(None, vol.Coerce(int)),
```

Comment:
Is it worth setting the defaults to max_int and min_int respectively? That way there's no need for None checking in check_boundaries.